src/compiler/compile/css/Stylesheet.ts

```diff
@@ -145,6 +145,8 @@ class Declaration {
 			? this.node.value.children[0]
 			: this.node.value;
 
+		if (first.type === 'Raw' && /^\s+$/.test(first.value)) return;
```

Comment:
I checked that the below statement is a syntax error. So this regex is ok.

```css
div {
  --foo:;
}
```

Reply:
@geoffrich Perhaps it would be useful to put a code comment here as to why we check this, since this applies specifically to chrome only.

Reply:
I added an explanatory comment, thanks for the suggestion!